core/java/iesi-core/src/test/java/io/metadew/iesi/metadata/configuration/action/DesignDatasetConfigurationTest.java

```diff
@@ -267,7 +268,7 @@ void actionDeleteFromScriptOnlyTest() {
         assertEquals(4, ActionParameterConfiguration.getInstance().getAll().size());
         assertEquals(Stream.of(action1, action2).collect(Collectors.toList()), ActionConfiguration.getInstance().getAll());
 
-        ActionConfiguration.getInstance().deleteByScript(new ScriptKey("1", 1));
+        ActionConfiguration.getInstance().deleteByScript(new ScriptKey("1", 1, LocalDateTime.now().toString()));
```

Comment:
Shouldn't this only work for "NA"?

Reply:
No, Actually we are deleting a record so we have to pass timestamp. 
If we will pass 'NA' then it will considered as non-deleted record.